lib/database/models/promo.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:objectbox/objectbox.dart';
+
+import 'coupon.dart' show Coupon;
+import 'resto.dart' show Resto;
+
+@Entity()
+class Promo {
```

Comment:
Maybe "Discount"? so we keep all the code in english.

Reply:
I changed it to Benefit

Reply:
Oh promo is the short of promotion. Sounds good!